.github/workflows/verifyHybridApp.yml

```diff
@@ -0,0 +1,148 @@
+name: Verify HybridApp build
+
+on:
+  workflow_call:
+  pull_request:
+    types: [opened, synchronize]
+    branches-ignore: [staging, production]
+    paths:
+      - '**.kt'
+      - '**.java'
+      - '**.swift'
+      - '**.mm'
+      - '**.h'
+      - '**.cpp'
+      - 'AndroidManifest.xml'
+      - 'project.pbxproj'
+      - 'package.json'
+
+concurrency:
+  group: ${{ github.ref == 'refs/heads/main' && format('{0}-{1}', github.ref, github.sha) || github.ref }}-verify-main
+  cancel-in-progress: true
+
+jobs:
+  verify_android:
+    name: Verify Android HybridApp builds on main
+    runs-on: ubuntu-latest-xl
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+          ref: ${{ github.event.pull_request.head.sha }}
+          token: ${{ secrets.OS_BOTIFY_TOKEN }}
+          # fetch-depth: 0 is required in order to fetch the correct submodule branch
+          fetch-depth: 0
+
+      - name: Update submodule to match main
+        run: |
+          git submodule update --init --remote
+          git fetch
+          git checkout main
+
+      - name: Configure MapBox SDK
+        run: ./scripts/setup-mapbox-sdk.sh ${{ secrets.MAPBOX_SDK_DOWNLOAD_TOKEN }}
+
+      - name: Setup Node
+        id: setup-node
+        uses: ./.github/actions/composite/setupNode
+        with:
+          IS_HYBRID_BUILD: 'true'
+
+      - name: Run grunt build
+        run: |
+          cd Mobile-Expensify
+          npm run grunt:build:shared
```

Comment:
Not sure ths is still necessary. But no worries, I will delete this [here](https://github.com/Expensify/App/issues/54877).